include/psim/fc/orbit_controller.hpp

```diff
@@ -42,6 +43,10 @@ class OrbitController : public OrbitControllerInterface<OrbitController> {
   gnc::OrbitControllerState _orbit_controller;
 
   Integer last_firing = 0;
+  Vector3 prev_dr_ecef=lin::nans<Vector3>();
+  Vector3 prev_dv_ecef=lin::nans<Vector3>();
+  Integer alpha=0.5;
+
 
```

Comment:
Also could just run clang format through here to cleanup the extra blank lines, equal sign spacing, etc.